WatchdogService/CleanupOperations.cs

```diff
@@ -270,140 +271,23 @@ internal async Task<int> EnumerateTableItemsAsync(CloudTable table)
 
             Stopwatch sw = Stopwatch.StartNew();
             ServiceEventSource.Current.Trace("EnumerateTableItemsAsync", table.Name);
+            TableClient tableClient = client.GetTableClient(table.Name);
```

Comment:
can be removed

Reply:
@christothes - I have updated, please review.